mozci/util/hgmo.py

```diff
@@ -38,7 +40,11 @@ def create(rev, branch="autoland"):
         return instance
 
     def _get_resource(self, url):
-        r = get_session("hgmo").get(url)
+        try:
+            r = get_session("hgmo").get(url)
+        except requests.exceptions.RetryError as e:
+            logger.warning(f"{e} error when getting {url}")
```

Comment:
Is the warning necessary here? It feels like it should be logged by a consumer in the event they decide to catch and ignore the exception (they can use `e.message` I believe).

Reply:
Agreed, removed.